xysched/README.md

```diff
@@ -46,12 +55,14 @@ future.Callback(fmt.Println, "increase x")
 xysched.Now() <- future
 ```
 
-3. Print a message every second.
+3.  Print a message every second.
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050498)
```suggestion
3. Print a message every second.
```